core/storage/merkle/polkadot_trie_db/polkadot_codec.hpp

```diff
@@ -18,38 +20,52 @@ namespace kagome::storage::merkle {
   class PolkadotCodec : public Codec {
```

Comment:
Please, write good docs to the methods of this class, some of them are kind of obfuscated